doc/contributing.md

```diff
@@ -1,16 +1,171 @@
+* A group of people collaborating to make the project better.
+
+This section of the guide describes how that system works.
+
+
+### PyViz
+
+The HoloViz project was previously named *PyViz* but the Python community suggested that it wasn't that appropriate and as such *PyViz* was renamed *HoloViz*, inspired by HoloViews. As renaming is a lot of work and can potentially break some systems, you will still find references to PyViz here and there. Notably, in the installation guide of each HoloViz package where the recommended installation command with conda is `conda install -c pyviz package`.
+
+*PyViz*, or [PyViz.org](https://pyviz.org/), is now a community project of its own. It is a fully open guide to all Python visualization tools. The HoloViz group maintains the [pyviz](https://github.com/pyviz) organization, but only for historical reasons, as it is meant to be shared by the whole Python visualization community.
+
+### GitHub/Git
+
+#### Organisations and repositories
+
+HoloViz consists of several Python projects. All these projects are version-controlled with [Git](https://git-scm.com/) and hosted on GitHub. Having all the projects hosted on GitHub means that HoloViz can rely on all the excellent features Github freely provides to open-source projects, including easy collaboration through *Issues and Pull Requests (PR)*, continuous integration (CI) with *GitHub Actions (GHA)* and documentation hosting with *GitHub Pages*.
+
+```{warning}
+It is crucial to keep in mind that nothing lasts forever and that, at some point, HoloViz may be forced to rely on other platforms or services. The more HoloViz relies on GitHub, the more difficult any transition to another system is.
+```
+
+The HoloViz group owns a few GitHub organizations:
+
+* [holoviz](https://github.com/holoviz/) is the main one where you are likely to contribute. It hosts the core packages maintained by the group.
+* [pyviz-dev](https://github.com/pyviz-dev/) hosts two main types of repositories:
+    * Packages that support maintaining the core HoloViz packages, including, for instance, `nbsite`, `nbsmoke`, `pyctdev`, `pyct` and `autover`. These support packages were developed when no alternative, or satisfying alternative, was available at the time the group needed them. Interestingly a few of these packages could have become pretty popular had they been more promoted. However, it was never a goal for the group that instead focused on improving data visualization with Python rather than improving the Python tooling.
+    * Repositories that are only used to host *dev* builds of the core packages websites, i.e., no actual work is done on these repositories. They just get updated automatically in a CI process. 
+* [pyviz-topics](https://github.com/pyviz-topics/) hosts repositories that demonstrate concrete usage of the HoloViz tools.
```

Comment:
I'm assuming there's probably too much friction to change the name of the `pyviz-dev` org to `holoviz-dev`, but what about changing `pyviz-topics` to `holoviz-topics` to make it consistent with its description?

Reply:
maybe we should also combine `holoviz-demos` and `pyviz-topics` as one org for holoviz uses curated by the holoviz team? I understand this is not the place to discuss these things, but just noting a thought to bring up later

Reply:
I'm not sure about the state of `holoviz-demos` to be honest and where it's really used. As for renaming the orgs, there's friction for sure but it's not impossible.